src/pages/mesh/basic/create-mesh.md

```diff
@@ -89,18 +89,13 @@ If you have an allowlist, consider adding the [edge mesh IP addresses](https://w
 
 The `aio api-mesh:create` response assigns you a `meshId`. Use the [`aio api-mesh:status`](../advanced/index.md#aio-api-meshstatus) command to see the status of your mesh creation. You can run the [`aio api-mesh:describe`](../advanced/index.md#aio-api-meshdescribe) command to get your `apiKey` and a GraphQL endpoint that you can use to query your mesh.
 
-After successfully running the status command, the response provides both legacy and edge URLs:
+After successfully running the status command, the response provides the edge URL:
```

Comment:
```suggestion
After successfully running the status command, the response provides the mesh URL:
```

On a side note, this message is a bit confusing. The `describe` command (rather than the `status` command) provides the mesh URL. However, the output below is that of the status command, wherein I don't see the URL. Not sure what the intention is here.